lib/isogram_cases.rb

```diff
@@ -13,7 +13,7 @@ def assertion
   end
 
   def skip
-    'skip' unless index.zero?
+    index.zero? ? '# skip' : 'skip'
```

Comment:
While we are here, we may as well make this `skipped` to match other implementations.  Keeping things as "standardized" as possible.  (Ref: Hamming)